### DISCUSSION TITLE
run main.ts

### DISCUSSION THREAD

**Discussion by patbi** [denoland/deno#10591] [Q&A] [answered]
**Hello Everyone I have this error for run this code**

import { App, Request, Response } from "https://deno.land/x/attain/mod.ts";

const app = new App();

app.listen({ port: 3500 });

**deno run --allow-net --unstable main.ts**

I work on windows10 64bits

Thank you.

![err](https://user-images.githubusercontent.com/19592278/117860404-2a4e4100-b288-11eb-989c-448f21cd0639.PNG)

**Comment by lucacasonato** [comment, marked as answer]
The `attain` library has not been updated for Deno 1.9.

**Reply by patbi** [reply]
Thank you @lucacasonato  ,and so how can I remedy this ?

**Reply by lucacasonato** [reply]
Do not use the `attain` library, or downgrade your Deno to a version that `attain` works on.

A good server library you might want to use is `oak`: https://deno.land/x/oak

**Reply by patbi** [reply]
Thank you.
